okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -431,10 +439,13 @@ public Void visit(QosException.Throttle exception) {
 
                 Optional<Duration> nonAdvertizedBackoff = backoffStrategy.nextBackoff();
                 if (!nonAdvertizedBackoff.isPresent()) {
-                    callback.onFailure(call, new SafeIoException(
-                            "Failed to complete the request due to QosException.Throttle",
-                            exception,
-                            UnsafeArg.of("requestUrl", call.request().url().toString())));
+                    callback.onFailure(
+                            call,
+                            new SafeIoException(
+                                    "Failed to complete the request due to QosException.Throttle",
+                                    exception,
+                                    UnsafeArg.of(
+                                            "requestUrl", call.request().url().toString())));
```

Comment:
not super convinced by this UnsafeArg breaking here... we're hitting the constraint of farthest right dots right?

Reply:
correct :(